src/colorizer/utils/data_utils.ts

```diff
@@ -327,3 +328,47 @@ export function getGlobalIdFromSegId(
 
   return rawGlobalId - 1; // -1 to convert to zero-based index
 }
+
+export function getLabelTypeFromParsedCsv(
+  headers: string[],
+  data: Record<string, string | undefined>[]
+): Map<string, LabelType> {
+  const labelTypeMap = new Map<string, LabelType>();
+  for (const header of headers) {
+    let hasIntegerValues = true;
+    let hasBooleanValues = true;
```

Comment:
would something like `isAllBooleans` and `isAllIntegers` describe these values more accurately? With the current names it's a bit confusing that these _start_ true.